qiskit/circuit/commutation_checker.py

```diff
@@ -29,7 +29,9 @@ def _identity_op(num_qubits):
 
 
 class CommutationChecker:
-    """This code is essentially copy-pasted from commutative_analysis.py.
+    """Checks if two Operations commute.
+
+    This code is essentially copy-pasted from commutative_analysis.py.
```

Comment:
We should probably make this line a code comment rather than public documentation haha.

Reply:
Heh yeah... I can do that instead. I at least added a line above to avoid this text being used as the summary in the TOC.